### ISSUE TITLE
Document timezone for timesheet importer (CEST timezone not accepted)

### ISSUE THREAD

**Issue by bash-stack** [kevinpapst/kimai2#2316]
SOLVED
===

## please give a hint in the docs that timezone HAS to be a php-intl string:

```
 Uncaught PHP Exception IntlException: "datefmt_create: time zone id 'CEST' extracted from ext/date DateTimeZone not recognized: U_ILLEGAL_ARGUMENT_ERROR"
 ```
--- 

Bug Grave:

**Describe the bug**
after a user accidentaly triggered `/team/timesheet/multi-delete`,
the CSV import suggested [in the official docs](https://www.kimai.org/documentation/imports.html#importing-timesheets) was used

The CSV had : 
* no rates set
* all the times set to exported 
* no tags as this gave :

  ```
  [ERROR] Failed importing timesheet row 2 with: An exception occurred while executing 'INSERT INTO kimai2_tags
  (name, color) VALUES (?, ?)' with  params ["xyzlalala", null]:
  SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'xyzlalala' for key 'UNIQ_27CAF54C5E237E06'
  ```

**To Reproduce**
1. Go to shell , do `bin/console kimai:import:timesheet --timezone=CEST --delimiter="," /tmp/reimport.csv`
2. be happy "oh it worked" for 10 seconds
2. Click on any imported timesheet entry
3. See error : 
![grafik](https://user-images.githubusercontent.com/62184749/107105550-44125f00-6827-11eb-8063-3fc2ff25a5a5.png)

( in logs :  "GET /en/team/timesheet/3456//edit HTTP/1.1" 500 )

**Logfile**
```
Add the last lines from your logfile at "var/log/prod.log" or "Doctor > Logs", around the time when the problem happened.
```

**Additional context**
 - Kimai version: 1.9 stable
 - PHP version: 7.4

**Comment by kevinpapst** [comment]
> user accidentaly triggered

Interesting feedback. Maybe this feature should be removed from the users list view. 

Thanks for the detailed feedback regarding the importer! Much appreciated, I update the documentation.

**Comment by bash-stack** [comment]
> > user accidentaly triggered
> 
> Interesting feedback. Maybe this feature should be removed from the users list view.
> 
> Thanks for the detailed feedback regarding the importer! Much appreciated, I update the documentation.

( Ja moin :D )

-> a separate feature-req

**Comment by kevinpapst** [comment]
👍 For the bug see #2318

**Comment by kevinpapst** [comment]
I added validation for the timezone.
And the tags are fixed as well.
Will be released with 1.15
